docs/reference/commandline/secret_create.md

```diff
@@ -34,32 +34,32 @@ command on a manager node.
 ### Create a secret
 
 ```bash
-$ cat secret.json | docker secret create -f - secret.json
+$ echo <secret> | docker secret create -f - my_secret
```

Comment:
Is the `-f -` required? I think that's a really strange syntax. Do any other Docker commands work like that?